### DISCUSSION TITLE
Issue installing with Yarp on both Ubuntu 18.04 and 22.04

### DISCUSSION THREAD

**Discussion by pamely** [robotology/community#651] [Q&A] [answered]
I have attempted to install YARP on my Ubuntu 18.04 device, however, I got an unexpected error message 

CMake Error at CMakeLists.txt:33 (find_package):
  Could not find a configuration file for package "YCM" that is compatible
  with requested version "0.15.1".

  The following configuration files were considered but not accepted:

    /usr/share/cmake/YCM/YCMConfig.cmake, version: 0.13.0

Kindly find attached a screenshot of the error as well as the error log file. 
Thank you in advance for your assistance. I am looking forward to reading from you soon. 

Pamely
![yarp-error](https://github.com/robotology/community/assets/15966026/558ffa1d-9df9-4dca-a9a6-4f33623edb0d)
[yarp_error.txt](https://github.com/robotology/community/files/12473656/yarp_error.txt)

**Comment by traversaro** [comment, marked as answer]
As it seems that `ycm` is installed in `/usr` install prefix, I guess you installed the `ycm-cmake-modules` package from apt, that is unfortunatly out of our control and outdated, i.e. it is too old to be used to build YARP.

I suggest to remove it, i.e. : 
~~~
sudo apt remove ycm-cmake-modules
~~~

and try again by installing from source a recent version of YCM, thanks!


fyi @drdanz

**Reply by pamely** [reply]
Thank you @traversaro. As you mentioned, the error is due to the fact that I installed `ycm-cmake-modules` pacakges from apt. To help news users of YARP can you please update the documentation available on https://www.yarp.it/latest//install_yarp_linux.html#install_on_linux . I had no clue that using apt was problematic. Many thanks for your quick reply.

**Reply by traversaro** [reply]
Thanks for the report, I opened an issue for this in https://github.com/robotology/yarp/issues/3009 .
